client/packages/sdk/src/price-estimation/__tests__/price-estimation.test.ts

```diff
@@ -0,0 +1,23 @@
+import { describe, expect, it } from "@jest/globals";
```

Comment:
This test is not mocked so each run will trigger API call to coingecko

Reply:
Fixed, refer to commit [659bacf](https://github.com/t3rn/t3rn/pull/1091/commits/659bacf98fbb5ec05ae91e992312e82db7cdaf79).